distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -415,13 +406,11 @@ private void cleanupCometDClients() {
     public void testFederatedQueryByWildCardSearchPhrase() throws Exception {
         String queryUrl = OPENSEARCH_PATH.getUrl() + "?q=*&format=xml&src=" + OPENSEARCH_SOURCE_ID;
 
-        // @formatter:off
```

Comment:
I think the formatter should be obeyed except in extreme circumstances. I don't think it's easy to read as is, so I'd vote to run the formatter